.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: java
 sudo: false
-script: mvn clean verify
+script: mvn clean verify -f tdt4140-gr1864/pom.xml $MAVEN_CLI_OPTS
 notifications:
```

Comment:
Travis CI should run all tests as well. Running `mvn clean install -f tdt4140-gr1864/pom.xml $MAVEN_CLI_OPTS` should do that.